stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Channel.kt

```diff
@@ -105,9 +105,14 @@ internal fun Channel.getLastMessagePreviewText(
 
         val attachments: SpannableString? = message.attachments
             .takeIf { it.isNotEmpty() }
-            ?.map { attachment ->
+            ?.mapNotNull { attachment ->
                 attachment.title?.let { title ->
-                    "${getAttachmentPrefix(context, attachment)} $title"
```

Comment:
`getAttachmentPrefix` has a nullable return type, inserting that into a string with a template could output the literal `"null"` string on the UI